terraform/modules/s3-bucket/main.tf

```diff
@@ -0,0 +1,15 @@
+resource "aws_s3_bucket" "s3_bucket" {
+  bucket = "${var.bucket_name}"
+  acl    = "private"
+  policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [{
+    "Effect": "Allow",
+    "Principal": {"AWS": "arn:aws:iam::${var.access_account_id}:root"},
+    "Action": "*",
+    "Resource": "arn:aws:s3:::${var.bucket_name}/*"
+  }]
+}
+EOF
```

Comment:
Yeah that makes sense. I don't actually know what heredoc is, was just copying other examples! I think you can use POLICY instead of EOF which would be better.